pyproject.toml

```diff
@@ -121,66 +121,68 @@ pinned = [
     "cbor2==5.4.6",
     "certifi==2023.5.7",
     "cffi==1.15.1",
-    "charset-normalizer==3.1.0",
-    "click==8.1.3",
-    "cryptography==40.0.2",
+    "charset-normalizer==3.2.0",
+    "click==8.1.5",
+    "cryptography==41.0.2",
     "cssselect2==0.7.0",
     "decorator==5.1.1",
-    "deepl==1.14.0",
+    "deepl==1.15.0",
     "Django==3.2.19",
     "django-cacheops==7.0.1",
-    "django-cors-headers==4.0.0",
-    "django-db-mutex==3.0.0",
+    "django-cors-headers==4.2.0",
+    "django-db-mutex==3.1.0",
     "django-debug-toolbar==4.1.0",
     "django-linkcheck==2.2.1",
     "django-polymorphic==3.1.0",
-    "django-redis==5.2.0",
+    "django-redis==5.3.0",
     "django-treebeard==4.7",
-    "django-webpack-loader==1.8.1",
+    "django-webpack-loader==2.0.1",
     "django-widget-tweaks==1.4.12",
     "executing==1.2.0",
     "feedparser==6.0.10",
-    "frozenlist==1.3.3",
+    "frozenlist==1.4.0",
     "funcy==1.18",
     "geographiclib==2.0",
     "geopy==2.3.0",
     "html5lib==1.1",
     "idna==3.4",
-    "ipython==8.13.2",
+    "ipython==8.14.0",
     "jedi==0.18.2",
-    "jsonschema==4.17.3",
-    "lxml==4.9.2",
+    "jsonschema==4.18.3",
+    "jsonschema-specifications==2023.6.1",
+    "lxml==4.9.3",
     "matplotlib-inline==0.1.6",
     "multidict==6.0.4",
-    "natsort==8.3.1",
+    "natsort==8.4.0",
     "oscrypto==1.3.0",
     "parso==0.8.3",
     "pexpect==4.8.0",
     "pickleshare==0.7.5",
-    "Pillow==9.5.0",
-    "prompt-toolkit==3.0.38",
+    "Pillow==10.0.0",
```

Comment:
The update of the pillow library seems to break the media upload:
```
  File "/home/integreat/integreat-cms/integreat_cms/cms/utils/media_utils.py", line 53, in generate_thumbnail
    image.thumbnail((size, size), resample=Image.ANTIALIAS)
                                           ^^^^^^^^^^^^^^^
AttributeError: module 'PIL.Image' has no attribute 'ANTIALIAS'
```

This is another example why #1997 is needed :sweat_smile:

Reply:
Ok, this was an easy fix :laughing: 
https://stackoverflow.com/questions/76616042/attributeerror-module-pil-image-has-no-attribute-antialias